config.json

```diff
@@ -1,9 +0,0 @@
-{
-  "appTitle" : "Address App",
-  "updateInterval" : 10000,
-  "simulateUnreliableNetwork" : true,
-  "currentLogLevel" : "DEBUG",
-  "specialLogLevels" : {
-    "SyncManager" : "TRACE"
-  }
-}
```

Comment:
No need for this file to be tracked as the default config file is created automatically. Not tracking it allows devs to have their own config.